src/compute-client/src/controller/instance.rs

```diff
@@ -1716,6 +1725,7 @@ where
             // tree to forward it on to the compute worker.
             otel_ctx,
             target: peek_target,
+            result_desc: intermediate_result_desc,
```

Comment:
Any reason to not call the `Peek` field `intermediate_result_desc` as well?

Reply:
See https://github.com/MaterializeInc/materialize/pull/32335#discussion_r2131888493